tools/peptideshaker/peptide_shaker.xml

```diff
@@ -7,6 +7,8 @@
     </macros>
     <requirements>
         <requirement type="package" version="1.1">peptide_shaker</requirement>
+        <!-- conda package - both packages should have the same content -->
+        <requirement type="package" version="1.1.3">peptide_shaker</requirement>
```

Comment:
Oh sorry, the name was wrong. It is peptide-shaker. With a dash.
If the version does not match 100% it will just take the latest version isn't it?